Modelica/Blocks/Tables.mo

```diff
@@ -156,7 +156,13 @@ extrapolation = 1: Hold the first or last value of the table,
                    (If smoothness is LinearSegments or ConstantSegments
                    this means to extrapolate linearly through the first/last
                    two table points.).
-              = 3: Periodically repeat the table data (periodical function).
+              = 3: Periodically repeat the table data (periodical function), 
+                   where the repetition period is table[end,1]-table[1,1].
+                   For u = table[end,1] the result is table[end,2]. However for the
+                   limit of u downwards to table[end,1] the result is table[<strong>1</strong>,2].
+                   This implies that for correct reproduction of a periodic function
+                   the first and last data values should be the same 
+                   (table[1,2] = table[end,2]).
```

Comment:
As motivated below for lines 436--442: The CombiTable1Ds possibly outputs an array for a single input (defined by the 'columns' parameter). Therefore the extension of '2' to '2:end' should be applied here as well.
```suggestion
              = 3: Periodically repeat the table data (periodical function), 
                   where the repetition period is table[end,1]-table[1,1].
                   For u = table[end,1] the output(-array) is table[end,columns]. However for the
                   limit of u downwards to table[end,1] the result is table[<strong>1</strong>,columns].
                   This implies that for correct reproduction of a periodic function
                   the first and last data values should be the same 
                   (table[1,2:end] = table[end,2:end]).
```